packages/vitest/src/node/cli.ts

```diff
@@ -167,6 +167,8 @@ function normalizeCliOptions(argv: CliOptions): CliOptions {
 }
 
 async function start(mode: VitestRunMode, cliFilters: string[], options: CliOptions): Promise<Vitest | undefined> {
+  process.title = 'node (vitest)'
```

Comment:
`process.title` can not be longer than the original title:

https://nodejs.org/api/process.html#processtitle

> on Linux and macOS, process.title is limited to the size of the binary name plus the length of the command-line arguments because setting the process.title overwrites the argv memory of the process

So I assume `+=` will likely crash node on affected platforms.

Reply:
I don't see how it will do that from the comment you linked.

Reply:
Or are you saying that `process.title` already fulfills these requirements before reassigning?

Reply:
From manual testing it looks like that title cannot be longer than `node {filename}` - in our case it's probably(?) `node vitest.mjs`.

I wasn't able to come up with the name that would not fit in the activity monitor. If you just have a simple file, the limit is reached pretty fast:

```js
process.title = 'node (vitest)'
setTimeout(() => {}, 5000)
```

```
node t.mjs
```

Activity monitor shows:

> node (vite

Reply:
> Activity monitor shows:
> 
> > node (vite

Would `vitest (node)` be better? 😄

Reply:
This problem exists only for this custom file due to the short name. Vitest doesn't have this issue